src/ontology/mappings/mondo_broadmatch_doid.sssom.tsv

```diff
@@ -7,14 +7,13 @@
 #   semapv: https://w3id.org/semapv/vocab/
 #   skos: http://www.w3.org/2004/02/skos/core#
 #   sssom: https://w3id.org/sssom/
-# license: https://creativecommons.org/publicdomain/zero/1.0/
+# license: https://w3id.org/sssom/license/unspecified
 # mapping_date: '2018-07-05'
 # mapping_set_id: http://purl.obolibrary.org/obo/mondo/mappings/mondo_broadmatch_doid.sssom.tsv
-# mapping_set_version: http://purl.obolibrary.org/obo/mondo/releases/2023-09-12/mappings/mondo_broadmatch_doid.sssom.tsv
+# mapping_set_version: http://purl.obolibrary.org/obo/mondo/releases/2023-12-07/mappings/mondo_broadmatch_doid.sssom.tsv
 # object_source: http://purl.obolibrary.org/obo/doid.owl
 # object_source_version: http://purl.obolibrary.org/obo/doid/releases/2018-07-05/doid.owl
-# subject_source: http://purl.obolibrary.org/obo/mondo.owl
-subject_id	subject_label	predicate_id	object_id	object_label	mapping_justification
-MONDO:0007744	cholesterol-ester transfer protein deficiency	skos:broadMatch	DOID:0111368	cholesterol-ester transfer protein deficiency	semapv:ManualMappingCuration
-MONDO:0009669	spinal muscular atrophy, type 1	skos:broadMatch	DOID:0060160	childhood spinal muscular atrophy	semapv:ManualMappingCuration
+subject_id	subject_label	predicate_id	object_id	mapping_justification
```

Comment:
why is `object_label` column removed?

Reply:
There is no `object_label` information in the obographs at all - you were somehow injecting them after the fact using merged.db or some such as far as I remember?

Reply:
Done.